src/sentry/grouping/enhancer/actions.py

```diff
@@ -60,12 +66,12 @@ def _from_config_structure(cls, val, version: int):
 
 
 class FlagAction(Action):
-    def __init__(self, key, flag, range):
+    def __init__(self, key: str, flag: bool, range: str) -> None:
         self.key = key
         self._is_updater = key in {"group", "app", "prefix", "sentinel"}
         self._is_modifier = key == "app"
         self.flag = flag
-        self.range = range
+        self.range = range  # e.g. "", "up", "down"
```

Comment:
These are the three possible values from `ACTION_FLAGS`. I will make a reference down below.